programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -955,7 +955,9 @@ static notification_message *create_notification_message(tunnel_identity *tnl_id
     notification->Identifier = strdup(tnl_id->Identifier);
     uv_timeval64_t now;
     uv_gettimeofday(&now);
-    notification->MfaTimeDuration = now.tv_sec - tnl_id->MfaLastUpdatedTime->tv_sec;
+    if(tnl_id->MfaLastUpdatedTime) {
```

Comment:
In general, should we test for `now.tv_sec - tnl_id->MfaLastUpdatedTime->tv_sec > 0`?

Reply:
I had commented out a line of code around Mfa and this segfaulted. it's not in this pr as I put those lines back but I wasn't expecting a segfault at the place i'd removed the code and it made me nervous that it might happen. That's why I added the check for MfaLastUpdatedTime to be "set" vs NULL